src/main/java/duke/Main.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import controller.MainWindow;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+/**
+ * Encapsulates main class that starts the application with duke and the user interface.
+ */
+public class Main extends Application {
+    private final Duke duke = new Duke();
```

Comment:
Maybe for duke can be in all caps to represent a constant.
" Constant names must be all uppercase using underscore to separate words."

Reply:
Thanks for point this out!